### PR TITLE
change ArticleCountMethod on lomithradienwiki per req

### PR DESCRIPTION
**Pull request by Reception123**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -124,6 +124,12 @@
 	'wgAutoConfirmCount' => array(
 		'default' => 10,
 	),
+	// Article Count Method
```
Comment (Southparkfan) [inline LocalSettings.php]: Please move this to "// Statistics"

File: LocalSettings.php [hunk 2]
```diff
@@ -124,6 +124,12 @@
...
+	'wgArticleCountMethod' => array(
+		'default' => false, // To update it, you will need to run the maintenance/updateArticleCount.php script
```
Comment (Southparkfan) [inline LocalSettings.php]: Default should be 'link'

File: LocalSettings.php [hunk 3]
```diff
@@ -124,6 +124,12 @@
...
+		'lomithradienwiki' => 'any',
+	),
+
```
Comment (Southparkfan) [inline LocalSettings.php]: Remove the unnecessary line

**Comment by Reception123** [comment]
changed

File: LocalSettings.php [hunk 4]
```diff
@@ -124,7 +124,6 @@
 	'wgAutoConfirmCount' => array(
 		'default' => 10,
 	),
-
```
Comment (Southparkfan) [inline LocalSettings.php]: Please restore this line

File: LocalSettings.php [hunk 5]
```diff
@@ -2456,7 +2455,11 @@
 	'wgSiteMatrixSites' => array(
 		'default' => array(),
 	),
-
+	// Statistics
+	'wgArticleCountMethod' => array(
+		'default' => 'link', // To update it, you will need to run the maintenance/updateArticleCount.php script
+		'lomithradienwiki' => 'any',
+	),
 	// Squid (aka Varnish)
```
Comment (Southparkfan) [inline LocalSettings.php]: Please add a line break between lines 2462 and 2463.

**Pull request merged**